docs/modules/ROOT/pages/how-tos/gke_node_maintenance.adoc

```diff
@@ -0,0 +1,36 @@
+= GKE Node Maintenance
+
+== Prerequisites
+
+* Access to the customer `gke-terraform` repository
+
+== Preparation
+
+* Login to the GKE console for the cluster you want to maintain / upgrade (in case you need to manually intervene)
+* Check out the customer's `gke-terraform` repository
+
+== Node maintenance
+
+GKE allows you to auto-upgrade your nodes. See https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-upgrades[Cluster upgrades] and https://cloud.google.com/kubernetes-engine/docs/how-to/node-auto-upgrades[Auto-upgrading nodes] for more details.
+Auto-Upgrade is enabled for the nodes by default.
+
+. Change Kubernetes Version
++
+No release channel is selected and therefore the version is set static. Change `cluster_version` in your `gke-terraform` repository.
+
+. Verify Auto-Upgrade Settings
++
+The node auto-upgrade is controlled by the following variables:
++
+* `maintenance_start_time`: Time window specified for recurring maintenance operations in RFC3339 format (default: `05:00`)
```

Comment:
```suggestion
* `maintenance_start_time`: Start time for the recurring maintenance operations in https://www.ietf.org/rfc/rfc5545.txt[RFC-5545] format (default: `05:00`)
```

According to the GKE docs, this is not an RFC3339 time, but a RFC5545 `DTSTART`. (https://cloud.google.com/kubernetes-engine/docs/how-to/maintenance-windows-and-exclusions).

Reply:
for the gcloud cli, yes, but api spec says RFC3339 See https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1/projects.locations.clusters#dailymaintenancewindow & https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1/projects.locations.clusters#recurringtimewindow & https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1/projects.locations.clusters#timewindow

I'll clarify and update the documentation to make it more clear